python/nammu/utils/__init__.py

```diff
@@ -313,21 +319,18 @@ def copy_yaml_to_home(jar_file_path, source_rel_path, target_path):
     it to ~/.nammu.
     '''
     try:
-        zf = zipfile.ZipFile(jar_file_path, 'r')
+        with zipfile.ZipFile(jar_file_path, 'r') as zf:
+            lst = zf.infolist()
```

Comment:
If `lst` is only used here, just move `zf.infolist()` into the for loop. e.g. `for zi in zf.infolist():`